proposals/js-types/Overview.md

```diff
@@ -145,6 +145,8 @@ This part of the proposal refines Wasm exported functions to have a suitable sub
 
 * The constructor for this class can be used to explicitly construct Wasm exported functions, closing a gap in the current API in that does not provide a way for JavaScript to put a plain JS function into a table (while the same is possible from inside Wasm).
 
+  * The constructor should not allow for passing a `WebAssembly.Function` object in, in such situations a `TypeError` should be raised.
```

Comment:
I think throwing is not the right resolution here, see my [latest comment](https://github.com/WebAssembly/js-types/issues/13#issuecomment-875621513) on #13.